integration/release_upgrade_linux_test.go

```diff
@@ -258,6 +258,97 @@ func getFileSize(t *testing.T, filePath string) int64 {
 	return st.Size()
 }
 
+func shouldManipulateContainersInPodAfterUpgrade(t *testing.T, criRuntimeService cri.RuntimeService, criImageService cri.ImageManagerService) upgradeVerifyCaseFunc {
+	var busyboxImage = images.Get(images.BusyBox)
+
+	t.Logf("Pulling image %q", busyboxImage)
+	_, err := criImageService.PullImage(&criruntime.ImageSpec{Image: busyboxImage}, nil, nil)
+	require.NoError(t, err)
+
+	t.Log("Create a sandbox")
+	sbConfig := PodSandboxConfig("sandbox", "running-pod")
+	sb, err := criRuntimeService.RunPodSandbox(sbConfig, "")
+	require.NoError(t, err)
+
+	t.Logf("Create a container config and run container in the pod")
+	containerConfig := ContainerConfig("running", busyboxImage, WithCommand("sleep", "1d"))
+	cn1InFirstSB, err := criRuntimeService.CreateContainer(sb, containerConfig, sbConfig)
```

Comment:
There is only one sandbox. `cn1` will be fine.

Reply:
done